.jenkins/pipelines/Azure/Nightly/libcxx_tests.Jenkinsfile

```diff
@@ -31,12 +31,12 @@ def ACCLibcxxTest(String label, String compiler, String build_type) {
 
 try {
     oe.emailJobStatus('STARTED')
-    parallel "Libcxx ACC2004 clang-8 Debug" :          { ACCLibcxxTest(FOCAL_LABEL, 'clang-8', 'Debug') },
-             "Libcxx ACC2004 clang-8 Release" :        { ACCLibcxxTest(FOCAL_LABEL, 'clang-8','Release') },
-             "Libcxx ACC2004 clang-8 RelWithDebInfo" : { ACCLibcxxTest(FOCAL_LABEL, 'clang-8', 'RelWithDebInfo') },
-             "Libcxx ACC1804 clang-8 Debug" :          { ACCLibcxxTest(BIONIC_LABEL, 'clang-8', 'Debug') },
-             "Libcxx ACC1804 clang-8 Release" :        { ACCLibcxxTest(BIONIC_LABEL, 'clang-8', 'Release') },
-             "Libcxx ACC1804 clang-8 RelWithDebInfo" : { ACCLibcxxTest(BIONIC_LABEL, 'clang-8', 'RelWithDebInfo') }
+    parallel "Libcxx ACC2004 clang-10 Debug" :          { ACCLibcxxTest(FOCAL_LABEL, 'clang-10', 'Debug') },
+             "Libcxx ACC2004 clang-10 Release" :        { ACCLibcxxTest(FOCAL_LABEL, 'clang-10','Release') },
+             "Libcxx ACC2004 clang-10 RelWithDebInfo" : { ACCLibcxxTest(FOCAL_LABEL, 'clang-10', 'RelWithDebInfo') },
+    parallel "Libcxx ACC1804 clang-10 Debug" :          { ACCLibcxxTest(BIONIC_LABEL, 'clang-10', 'Debug') },
```

Comment:
This 2nd `parallel` would cause the ACC1804 runs in series after ACC2004. Is this part of the proposal?

Reply:
I think this was overlooked during a merge

Reply:
Ok, let's remove the extra one on line 37 :)